### ISSUE TITLE
Configure name of dll from BitMono CLI

### ISSUE THREAD

**Issue by techei** [sunnamed434/BitMono#109]
### Summary

While obfuscating a dll BitMono CLI takes 3 arguments as of now:
::-f dll to obfuscate
::-l location of all dependencies
::-o Path to store obfuscated dll

It will be helpful feature if output DLL name is also configurable here.

### Basic Example

Now BitMono created a DLL with same name, if output path is same as original DLL path this feature will be useful.

**Comment by sunnamed434** [comment]
Yeah, it would be cool to add, do you have any ideas about the argument name?

**Comment by techei** [comment]
I have 2 ideas, we can use below 2 as argument name:
1. -o: path can be folder path + file name (if this argument ends with \ it's output path otherwise it's path + fileName)
2. -on: Output dll(/exe) name

**Comment by techei** [comment]
I will raise a PR over this weekend(on 04/01 - 04/02), if no one else is taking care of this enhancement.

**Comment by sunnamed434** [comment]
> I will raise a PR over this weekend(on 04/01 - 04/02), if no one else is taking care of this enhancement.

@techei Do you still plan to add this?
